docs/email/postfix/postfix-spf-dkim-debian-8.md

```diff
@@ -0,0 +1,375 @@
+
+The instructions for setting up DNS for SPF and DKIM are generic. The instructions for configuring hooking the SPF policy agent and OpenDKIM into Postfix should work on any distribution with adjustments for the package tool and the exact path to OpenDKIM's socket file in Postfix.
+
+{: .note}
+>
+>The steps required in this guide require root privileges. Be sure to run the steps below as **root** or with the `sudo` prefix. For more information on privileges see our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
+
+{: .note}
+>
+>You must already have Postfix installed, configured and working. Refer to the [Linode Postfix Guides](https://www.linode.com/docs/email/postfix/) for assistance.
+
+{: .caution}
+>
+>Publishing an SPF DNS record without having the SPF policy agent configured within Postfix is safe, however publishing DKIM DNS records without having OpenDKIM working correctly within Postfix can result in your mail being discarded by the receiver's mail server.
+
+Installing required packages
+----------------------------
+
+You'll need to switch to a root shell using `su -` or `sudo -s` to install packages.
+
+1. Install the required packages with this command:
+
+    apt-get install opendkim opendkim-tools postfix-policyd-spf-python
+
+Tell it "Y" when it asks whether to download and install the listed packages. It won't take long and won't prompt for anything.
+
+2. Add user `postfix` to the `opendkim` group so that Postfix can access OpenDKIM's socket when it needs to:
+
+    adduser postfix opendkim
+
+ Remember to exit your root shell when you're done.
+
+Setting up SPF
+--------------
+
+## Adding SPF records to DNS ##
+
+The value in an SPF DNS record will look something like the following examples. The full syntax is at [the SPF record syntax page](http://www.openspf.org/SPF_Record_Syntax).
+
+1. Allow mail from all hosts listed in the MX records for the domain:
+
+    "v=spf1 mx -all"
+
+2. Allow mail from a specific host:
+
+    "v=spf1 a:mail.example.com -all"
+
+The "v=spf1" tag is required and has to be the first tag. The last tag, "-all", indicates that mail from your domain should only come from servers identified in the SPF string, anything coming from any other source is forging your domain. An alternative is "~all", indicating the same thing but also indicating that mailservers should accept the message and flag it as forged instead of rejecting it outright. I prefer "-all" to make it harder for spammers to forge my domain successfully, but if you're nervous you might want to use "~all" so no mail gets lost because of errors.
```

Comment:
The last sentence needs to be rewritten to remove the fist person, and make it less a personal opinion and more a recommendation.